src/resmom_win/start_exec.c

```diff
@@ -2179,7 +2224,7 @@ finish_exec(job *pjob)
 				free(argv_out[0]);
 				argv_out[0] = progname_out;
 				argv_str = str_array_to_str(argv_out,
-								" ");
+								' ');
```

Comment:
Add this to the end of the previous line.

Reply:
Done.